GVRf/Framework/framework/src/main/jni/vulkan/vulkanCore.h

```diff
@@ -195,6 +206,7 @@ class VulkanCore {
     VkSurfaceKHR m_surface;
 
     std::vector<VkCommandBuffer*> swapChainCmdBuffer;
+    std::vector<VkCommandBuffer*> postEffectCmdBuffer;
 
```

Comment:
do you need multiple cmd buffers for posteffect? if you are waiting to finish rendering, you dont need multple.